packages/enketo-transformer/test/markdown.spec.ts

```diff
@@ -68,6 +68,18 @@ describe('markdown', () => {
                 '[link](http://example.org)',
                 '<a href="http://example.org" rel="noopener" target="_blank">link</a>',
             ],
+            [
+                '[link](http://example.org/cgi-bin/Markdown.pl?a=1&b=2)',
+                '<a href="http://example.org/cgi-bin/Markdown.pl?a=1&b=2" rel="noopener" target="_blank">link</a>',
+            ],
+            [
+                '[link with a "quote" inside](http://example.org)',
```

Comment:
Did this case intend to have a quote inside?

Reply:
only in the link text... would `link text with "quotes" inside` be clearer?

Reply:
The intent is to verify that the label is not uri-encoded?